metricbeat/module/elasticsearch/elasticsearch_integration_test.go

```diff
@@ -110,31 +121,110 @@ func getConfig(metricset string) map[string]interface{} {
 func createIndex(host string) error {
 	client := &http.Client{}
 
-	resp, err := http.Get("http://" + host + "/testindex")
+	if checkExists("http://" + host + "/testindex") {
+		return nil
+	}
+
+	req, err := http.NewRequest("PUT", "http://"+host+"/testindex", nil)
 	if err != nil {
 		return err
 	}
-	resp.Body.Close()
 
-	// This means index already exists
-	if resp.StatusCode == 200 {
+	resp, err := client.Do(req)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != 200 {
+		return fmt.Errorf("HTTP error %d: %s", resp.StatusCode, resp.Status)
+	}
+
+	return nil
+}
+
+// createIndex creates and elasticsearch index in case it does not exit yet
+func enableLicense(host string) error {
```

Comment:
Nit: Name this `enableTrialLicense` just to be more specific about what it actually does?